app/controllers/SubmitFormController.scala

```diff
@@ -225,18 +225,18 @@ class SubmitFormController @Inject()(mcc: MessagesControllerComponents,
                   .addingToSession(SessionKeys.returnData -> Json.toJson(sessionModel).toString())
                   .removingFromSession(SessionKeys.viewModel)
               } else {
-                logger.warn(s"[SubmitFormController][submitSuccess] Obligation end date for period $periodKey has not yet passed.")
+                errorLog(s"[SubmitFormController][submitSuccess] Obligation end date for period $periodKey has not yet passed.")
                 errorHandler.showBadRequestError
               }
             case _ =>
-              logger.warn(s"[SubmitFormController][submitSuccess]: Obligation not found for period key $periodKey")
+              warnLog(s"[SubmitFormController][submitSuccess]: Obligation not found for period key $periodKey")
               Redirect(appConfig.returnDeadlinesUrl)
           }
         case (Left(error), _) =>
-          logger.warn(s"[SubmitFormController][submitSuccess] Error received when retrieving customer details $error")
+          errorLog(s"[SubmitFormController][submitSuccess] Error received when retrieving customer details $error")
           errorHandler.showInternalServerError
         case (_, Left(error)) =>
-          logger.warn(s"[SubmitFormController][submitSuccess] Error received when retrieving obligation details $error")
```

Comment:
same here

Reply:
i think this should be kept as errorLog since it throws an internal server error when it cant retrieve the obligations details